generators/init/templates/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Need to add to dependabot and load it.
https://github.com/jhipster/generator-jhipster/blob/09f1869ccd58ddd5f9d0b6db45de04df5b205b14/generators/client/index.js#L224-L236